public/style/styles-compiler.scss

```diff
@@ -6,13 +6,12 @@
 'base/forms';
 
 @import 'components/animation',
-'components/btn',
+'components/buttons',
```

Comment:
Zurb's foundation and Boostrap both have `buttons` instead of `btn`. I like the clarification.